docs/policies.rst

```diff
@@ -0,0 +1,34 @@
+========
+Policies
+========
+
+Policy interfaces
+=================
+
+.. autoclass:: chainerrl.policy.Policy
+   :members:
```

Comment:
```suggestion
   :members:

   .. automethod:: __call__
```

Reply:
Can you confirm that `__call__` is documented?

Reply:
Will do.

Reply:
It is: https://github.com/chainer/chainerrl/blob/master/chainerrl/policy.py#L22

Perhaps there's a formatting issue with the docstring? I'll take a look.

Reply:
Sorry, what I meant is, can you build a doc and confirm `__call__` is documented on the built html?

Reply:
Ah ok. I had actually done that when you made your original comment (https://github.com/chainer/chainerrl/pull/516#issuecomment-519874332) and it wasn't showing up. I'll revisit this issue.

Reply:
Suggested change works: 
![policy_fix](https://user-images.githubusercontent.com/10005453/63329960-d8f07680-c36d-11e9-8dc9-ee4aa6c425b6.png)